### PR TITLE
browser(webkit): add ip address to Network.responseReceived

### PR DESCRIPTION
**Pull request by JoelEinbinder**
For #6624﻿

### CODE REVIEW

File: browser_patches/webkit/patches/bootstrap.diff [hunk 1]
```diff
@@ -727,10 +727,20 @@ index 0000000000000000000000000000000000000000..587287d52fde2735cbae34a27a0f673b
...
 --- a/Source/JavaScriptCore/inspector/protocol/Network.json
 +++ b/Source/JavaScriptCore/inspector/protocol/Network.json
-@@ -321,6 +321,13 @@
+@@ -77,7 +77,8 @@
+                 { "name": "source", "type": "string", "enum": ["unknown", "network", "memory-cache", "disk-cache", "service-worker", "inspector-override"], "description": "Specifies where the response came from." },
+                 { "name": "requestHeaders", "$ref": "Headers", "optional": true, "description": "Refined HTTP request headers that were actually transmitted over the network." },
+                 { "name": "timing", "$ref": "ResourceTiming", "optional": true, "description": "Timing information for the given request." },
+-                { "name": "security", "$ref": "Security.Security", "optional": true, "description": "The security information for the given request." }
++                { "name": "security", "$ref": "Security.Security", "optional": true, "description": "The security information for the given request." },
++                { "name": "address", "type": "string", "optional": true, "description": "Remote IP address and port." }
```
Comment (rwoll) [inline browser_patches/webkit/patches/bootstrap.diff:739]: If it's convenient, lets return `serverIPaddress::string` and `port::number` in distinct fields to match CR and FF.

Comment (JoelEinbinder) [reply browser_patches/webkit/patches/bootstrap.diff:739]: It's not convenient!

File: browser_patches/webkit/patches/bootstrap.diff [hunk 2]
```diff
@@ -3189,7 +3199,17 @@ index 67831035d9d3dd86807802e89455e9a0776c36ea..c3cb206baa21ccb1cdf4b9f0e69d5464
      if (resourceLoader)
          responseObject->setTiming(buildObjectForTiming(response.deprecatedNetworkLoadMetricsOrNull(), *resourceLoader));
  
-@@ -488,8 +492,14 @@ static InspectorPageAgent::ResourceType resourceTypeForLoadType(InspectorInstrum
+@@ -386,6 +390,9 @@ RefPtr<Protocol::Network::Response> InspectorNetworkAgent::buildObjectForResourc
+ 
+             securityPayload->setCertificate(WTFMove(certificatePayload));
+         }
++        
++        if (auto networkLoadMetrics = response.deprecatedNetworkLoadMetricsOrNull())
```
Comment (pavelfeldman) [inline browser_patches/webkit/patches/bootstrap.diff:3207]: fyi: I vaguely remember that it did not work on a mac.

**Review by pavelfeldman — APPROVED** [review]

**Comment by JoelEinbinder** [comment]
It turns out this data is already in Network.loadingFinished

**Comment by rwoll** [comment]
I see it in the Protocol.Newtork.Metrics section nested in Network.loadingFinished, so I'll pull from there.

**Pull request closed without merging**